michaelogala/shot
# Ft url shortener

# #129655283 BUILD URL SHORTENING SYSTEM

A combination of a normalized database, controllers and models was used to set up a system for receiving long urls and producing a shortened version. Also a redirect system was built that causes all shortened urls to redirect to the original url.
## MANUAL TESTING

Visit the root path (localhost:3001), paste your long url in the textfield thats centered on the page and click on the shorten button. The shortened url will should be displayed in a green alert box.

![screen shot 2016-09-08 at 10 07 25 pm](https://cloud.githubusercontent.com/assets/20759355/18367054/2b6b061c-7611-11e6-8294-bc916e3bb894.png)

![screen shot 2016-09-08 at 10 07 42 pm](https://cloud.githubusercontent.com/assets/20759355/18367071/3abf6d7e-7611-11e6-90d8-26ab3a41e674.png)
